HuntTheWumpus/HuntTheWumpus/Map.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace HuntTheWumpus
+{
+    public static class Map
+    {
+        private static Random _random = new Random();
+        public static List<Room> Rooms { get; set; } = new List<Room>(21);
+
```

Comment:
Suggestion for future: Read connected rooms from outside text file and loop through the add rooms statement or even store them in an array.